### ISSUE TITLE
Double tap accessibility action doesn't work

### ISSUE THREAD

**Issue by 532910** [mozilla-mobile/fenix#13334]
## Steps to reproduce
1. Install end setup something like [Native-ClipBoard](https://github.com/DHM47/Native-Clip-Board) that will register double tap accessibility action

2. Double tap the address field

### Expected behavior
Accessibility action should be executed.

### Actual behavior
Just nothing happens.

### Device information

* Android device: 9
* Fenix version: the latest one and all above

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-4439)

**Comment by 532910** [comment]
I believe the stale fact is not a reason to close issues.

**Comment by 532910** [comment]
I believe the stale fact is not a reason to close issues.

**Comment by 532910** [comment]
still actual
